lib/opentok.js

```diff
@@ -524,13 +605,101 @@ OpenTok = function (apiKey, apiSecret, env) {
     }
     else {
       options.sessionId = sessionId;
+      if (!options.streamMode) {
+        options.streamMode = 'auto';
+      }
       client.startBroadcast(options, function (err, json) {
         if (err) { return callback(new Error('Failed to start broadcast. ' + err)); }
         return callback(null, new Broadcast(client, json));
       });
     }
   };
 
+  /**
+  * Adds a stream to a broadcast that has the streamMode set to manual.
```

Comment:
```suggestion
```suggestion
  * Adds a stream to a broadcast that has the streamMode set to manual.
  * You can call the method repeatedly with the same stream ID, to toggle
  * the stream's audio or video in the broadcast.
```